CRM/CiviAwards/Setup/AddAwardsCgExtendsOptionValue.php

```diff
@@ -33,4 +36,28 @@ public function apply() {
     ]);
   }
 
+  /**
+   * Enables/Disables the Awards option values
+   *
+   * @param $newStatus
```

Comment:
Missing parameter comment.

You need to run npm install in the civiawards root so that the php linter is added to git pre-commit hook. That way you get notified of things like this before you commit.